Summer 2024 updates (#4735)
Couchers-org/couchers: app/web/markdown/roadmap.md

---
title: Roadmap
crumb: Roadmap
type: general
description: Keep up to date with the Couchers development roadmap.
---

*This page was last updated 7th September, 2024.*

# Our current milestone: a Safe & Active Community (Q3 2024)

We are currently (Q3 2024, July to September) focussing on building around the goal of a *Safe & Active community*.

This milestone will include efforts such as:

* rebuilding the map search to filter and sort members better, better highlight active members, and show more information on the map,

> * building a system of "activeness probes" to keep track of who is still interested in hosting Couchers,

Reviewer: ```suggestion
* building a system of "activeness probes" to keep track of who is still interested in hosting Couchers (eg. allowing inactive users to be "filtered out"),
```

* rolling out the first version of Strong Verification to help build safety and trust on the platform,

> * limiting sending requests and some other features to only those with completed profiles to reduce junk requests,

Reviewer: ```suggestion
* limiting sending requests and some other features to only those with completed profiles to reduce empty profiles and low-quality requests,
```

* completing the work on push notifications and the new notification system, and
* migrating the old handbook to the new help center to make information more easily accessible.

# Detailed roadmap of current development work

You can find out up-to-date and in-depth information about what we are currently working on on our [Development Roadmap view on Github](https://github.com/orgs/Couchers-org/projects/4/views/13) which we update regularly as we plan our day-to-day development work. Note that some of the information there may be highly technical in nature and not be accessible to all audiences; and dates or timelines listed there are by no means guaranteed and are only an indication. Please keep in mind that Couchers is a volunteer project built, maintained, and operated entirely by volunteers in their free time!

# How we structure our strategic and development goals and roadmap

> We currently build around a quarter-based system of strategic *milestones*. We divide the year into calendar quarters, Q1 (Jan to Mar) to Q4 (Oct to Dec) and choose a strategic and development focus that we call a *milestone* for each quarter. This helps the various teams at Couchers to rally behind a common goal and have clarity on what they should be working on together. Each strategic milestone is brainstormed and chosen by the [Board of Directors](/foundation) along with the [volunteer team](/team).

Reviewer: ```suggestion
We currently build around a quarter-based system of strategic *milestones*. We divide the year into calendar quarters, Q1 (Jan to Mar) to Q4 (Oct to Dec) and choose a strategic and development focus that we call a *milestone* for each quarter. This helps the various teams at Couchers to rally behind a common goal and have clarity on what they should be working on together. Each strategic milestone is brainstormed and chosen by the [Board of Directors](/foundation) along with the [volunteer team](/team) using input from the community.
```
